### ISSUE TITLE
Switch from card view to event list view in the List all facilities page

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#62]
## User story

1. As a facility manager
2. I want to have all facilities listed as an event list instead of card view
3. So that I can have a better overview of all facilities

## Acceptance criteria

- [x] All facilities are listed as a list using the event list view
- [x] Developers create at least 20 facilities to provide a similar view as the real life use case
- [x] The view for facilities with errors can stay as a card view (giving only 2,3 facilities will have errors)

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x] Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release
